index.html

```diff
@@ -35,6 +35,9 @@ <h2 id="weather-text">Fetching Weather...</h2>
         </div>
         <div id="other-content">
         </div>
+        <svg viewBox="0 0 24 24" id="settings-cog" style="width:48px;height:48px; position: absolute; bottom: 25px; right: 25px;">
```

Comment:
@reesericci Since you added the style to `main.css`, I think you can remove the `style` attribute?

Reply:
lemme edit to fix this

Reply:
done